driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -144,7 +143,12 @@ impl PriceFinding for LinearOptimisationPriceFinder {
         orders: &[models::Order],
         state: &models::AccountState,
     ) -> Result<models::Solution, PriceFindingError> {
-        let token_ids: Vec<String> = (0..models::TOKENS).map(token_id).collect();
+        let max_token_id = orders
```

Comment:
I think we can omit the tokens that don't have any orders. E.g. if we only have an order trading token0 for token7 the list can be [token0, token7] instead of [token0, token1, ...]

Reply:
Yes, I like this idea also and will work toward making this a reality! Looking forward to it!

Reply:
Did this happen?

Reply:
I was saving this for part 3 which is essentially the content of #412